src/main/java/com/courtcanva/ccfranchise/services/FranchiseeService.java

```diff
@@ -0,0 +1,37 @@
+package com.courtcanva.ccfranchise.services;
+
+import com.courtcanva.ccfranchise.dtos.FranchiseeAndStaffGetDto;
+import com.courtcanva.ccfranchise.dtos.FranchiseeAndStaffPostDto;
+import com.courtcanva.ccfranchise.dtos.StaffGetDto;
+import com.courtcanva.ccfranchise.mappers.FranchiseeMapper;
+import com.courtcanva.ccfranchise.models.Franchisee;
+import com.courtcanva.ccfranchise.repositories.FranchiseeRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class FranchiseeService {
+    private final FranchiseeRepository franchiseeRepository;
+
+    private final FranchiseeMapper franchiseeMapper;
+
+    private final StaffService staffService;
+
+    @Transactional
+    public FranchiseeAndStaffGetDto createFranchiseeAndStaff(FranchiseeAndStaffPostDto franchiseeAndStaffPostDto) {
+
+        Franchisee franchisee = franchiseeRepository
+                .save(franchiseeMapper.franchiseePostDtoToFranchisee(franchiseeAndStaffPostDto.getFranchiseePostDto()));
+        franchisee.setIsVerified(false);
```

Comment:
can we set default value of this field as `fasle`?  Then we don't need to set here.

Reply:
I set the default value to false for franchisee and staff.